web/modules/custom/joinup_notification/config/install/message.template.rdf_entity_distribution_delete.yml

```diff
@@ -0,0 +1,19 @@
+langcode: en
+status: true
+dependencies: {  }
+template: rdf_entity_distribution_delete
+label: 'Rdf entity - asset distribution - delete'
+description: 'Message to the owner when a distribution is deleted'
+text:
+  -
+    value: "<p>Joinup - Content has been deleted</p>\r\n"
```

Comment:
Change "content" with "distribution", so at least when we receive an e-mail we understand what is the e-mail for.
And sneak in also the same change on the solution delete mail 😄